cmd/common.go

```diff
@@ -139,6 +139,18 @@ func listRoleTemplateBindings(ctx *cli.Context, b []RoleTemplateBinding) error {
 	return writer.Err()
 }
 
+// serverCurrent command to display the name of the current server in the local config
+func serverCurrent(ctx *cli.Context) (string, string, error) {
```

Comment:
nit: I think this should be renamed to
```suggestion
func getCurrentServerFromConfig(ctx *cli.Context) (string, string, error) {
```

`getCurrentServer` or `currentServer` would be okay as well.